packages/editor-ui/src/router.ts

```diff
@@ -130,6 +130,15 @@ export const routes = [
 			},
 			middleware: ['authenticated'],
 		},
+		beforeEnter: (to, _from, next) => {
+			const templatesStore = useTemplatesStore();
+			if (!templatesStore.hasCustomTemplatesHost) {
+				const id = Array.isArray(to.params.id) ? to.params.id[0] : to.params.id;
+				window.location.href = templatesStore.getWebsiteTemplatePageURL(id);
```

Comment:
does this have to open in the same window? would not it be better if it opened in a new tab?

Reply:
This should trigger only when users land on one of templates routes directly, in that case I don't think it makes sense to open website in new tab. Clicking sidebar item and node creator links opens new tabs.

Reply:
ah I see. that makes sense. I misunderstood.